gallery/src/styles/NxPageTitle/NxPageTitlePage.tsx

```diff
@@ -60,11 +65,29 @@ const NxPageTitlePage = () =>
       </table>
     </GalleryDescriptionTile>
 
-    <RawHtmlExample html={nxPageTitleCode} />
-    <CodeExample content={nxPageTitleCode} />
+    <GalleryExampleTile title="NX Page Title Example"
+                        id="nx-page-title-example"
+                        defaultCheckeredBackground={true}
+                        htmlExample={nxPageTitleCode}
+                        codeExamples={nxPageTitleCode}>
+      A simple example of an <code className="nx-code">nx-page-title</code>.
+    </GalleryExampleTile>
+
+    <GalleryExampleTile title="NX Page Title with Actions Example"
+                        id="nx-page-title-actions-example"
+                        defaultCheckeredBackground={true}
+                        htmlExample={nxPageTitleActionsCode}
+                        codeExamples={nxPageTitleActionsCode}>
+      An example of <code className="nx-code">nx-page-title</code> with actions in the form of a drop-down.
```

Comment:
Why is this change showing up in this PR? This is separate and already merged to master, right?

Reply:
> Why is this change showing up in this PR? This is separate and already merged to master, right?

When I added the new example I updated the two other examples to use the new gallery example tile